### PR TITLE
[Fix][Connector-V2] Fix When reading Excel data, string and date type conversion errors

### PR DESCRIPTION
**Pull request by CosmosNi**
#7759 

handler  string and date type convert
image
image

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
This patch cannot cover all situations. We should ensure from the source side that no abnormal data is sent to the downstream.

**Comment by CosmosNi** [comment]
@Hisoka-X When converting the type, it is not converted to String

File: seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/ExcelReadStrategy.java [hunk 1]
```diff
@@ -215,7 +215,7 @@ private Object convert(Object field, SeaTunnelDataType<?> fieldType) {
             case ARRAY:
                 return objectMapper.readValue((String) field, fieldType.getTypeClass());
             case STRING:
-                return field;
+                return String.valueOf(field);
```
Comment (Hisoka-X) [inline seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/reader/ExcelReadStrategy.java:216]: Thanks @CosmosNi , please add test case in https://github.com/apache/seatunnel/tree/c685c5c71dd29618859e2f52fe552333b00729e8/seatunnel-connectors-v2/connector-file/connector-file-base/src/test/java/org/apache/seatunnel/connectors/seatunnel/file/writer

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @CosmosNi

**Review by liugddx — APPROVED** [review]

**Pull request merged**